src/axom/sidre/examples/sidre_mfem_datacollection_restart.cpp

```diff
@@ -204,11 +204,11 @@ int main(int argc, char* argv[])
   for(int i = 0; i < n_iter; i++)
   {
     sim_state.step(dt);
-
-    // then save it at each iteration
-    dc.Save("sidre_mfem_datacoll_restart_ex", "sidre_hdf5");
   }
 
+  // then save it at the end of the simulation
+  dc.Save("sidre_mfem_datacoll_restart_ex", "sidre_hdf5");
```

Comment:
Any particular reason for this change?

Perhaps you can dump a checkpoint every  `n_checkpoint` steps?  ( Where `0 < n_checkpoint < n_iters` )